features/step_definitions/direct_award_steps.rb

```diff
@@ -7,3 +7,69 @@
     And I click 'Save and continue'
   }
 end
+
+When (/^I have created and ended a search called '(.*)'$/) do |search_name|
+  steps %{
+    And I have created and saved a search called '#{search_name}'
+    And I am on the /buyers page
+    Then I click the 'View your saved searches' link
+    Then I click the '#{search_name}' link
+    Then I am on the '#{search_name}' page
+    And I click the 'End search' link
+    Then I am on the 'End your search' page
+    And I click the 'End search and continue' button
+    Then I am on the '#{search_name}' page
+  }
+end
+
+When (/^I have downloaded the search results as a file of type '(.*)'$/) do |file_type|
+  steps %{
+    And I click the 'Download search results' link
+    And I am on the 'Download your search results' page
+    And I click the 'Download search results as a spreadsheet' link
```

Comment:
This step purports to let you define the filetype you want to download, but only downloads it as a spreadsheet, so when you reference the parent step you'll always need to pass in `ods`, won't you?

Reply:
You are correct. I have made the change so that it now allows filetype to be defined and downloads the appropriate file type